Client/src/components/common/date-picker/index.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./style.css";
+const days = ["Sat-Mon-Wedn", "Sun-Tues-Thurs"];
+function DatePicker() {
+  return (
+    <select className="select">
```

Comment:
how you will get the value from the input. 
you need to an event handler, to get the value from the input. 
Article [https://reactjs.org/docs/forms.html](url) Enjoy.